aif360/metrics/mdss/ScoringFunctions/BerkJones.py

```diff
@@ -0,0 +1,87 @@
+from aif360.metrics.mdss.ScoringFunctions.ScoringFunction import ScoringFunction
+from aif360.metrics.mdss.ScoringFunctions import optim
+
+import numpy as np
+
+class BerkJones(ScoringFunction):
+
+    def __init__(self, **kwargs):
+
+        super(BerkJones, self).__init__()
```

Comment:
Here and in Bernoulli functions, more comments can be added in the docstring to make it a bit more elaborate. For example what is q?

Reply:
Can you also provide some reference to Berk Jones?

Reply:
What are the situations in which Berk Jones vs Bernoulli vs Poisson scores are used?

Reply:
I've included some more comments in the scoring functions for help elaborate and provide references. In, general Poisson and Bernoulli are used when the user wishes to make parametric assumptions about the outcome whereas Berk-Jones is non-parametric scan statistic.